src/commands/utility/IDCommand.js

```diff
@@ -30,10 +30,18 @@ class IDCommand extends Command {
         const embed = new Discord.MessageEmbed()
             .setTitle(`User search for ${fullName}`);
         if (users.size === 0) {
-            embed.setDescription("No users found");
-            embed.setColor(util.color.red);
+            embed.setDescription("No users found")
+                .setColor(util.color.red);
             return await this.message.channel.send(embed);
         }
+        if (users.size > 150) {
+            embed.setTitle(`First 150 results of user search for ${fullName}`)
+            let i = 0;
+            users = users.filter(() => {
```

Comment:
This will loop through all users in the collection, even though we could just stop after the first 150 (not a big issue, but not ideal).
A for-loop that breaks if i >= 150 might be a better solution.